lib/omniauth_keycloak/configuration.rb

```diff
@@ -52,7 +52,13 @@ def discovery_url
   end # #discovery_url
 
   def discovery_object
-    ::HTTParty.get(discovery_url) || {}
+    begin
+      ::HTTParty.get(discovery_url) || {}
+    rescue StandardError => e
+      Rails.logger.error e
+      puts '[OmniauthKeycloak] auth-server-url is not a valid url'
+      return "#{self.realm_url}/.well-known/openid-configuration"
```

Comment:
hm, nope, i guess thats wrong,
the method implicite returns an object, so returning the duscovery url would be incorrect